deps/checksums/unwind

```diff
@@ -1,3 +1,5 @@
+LibOSXUnwind.v0.0.6+1.aarch64-apple-darwin.tar.gz/md5/1d68335f3b893ee39d91e670cf1a9144
+LibOSXUnwind.v0.0.6+1.aarch64-apple-darwin.tar.gz/sha512/a3a4d8a4cf819bab09660c474849ca03a02d0c1c3e49bb4c92f25d974c827c0f932116a56d073cf20a9a75af19ee122aebc67751e9b9acd814ef3eb43371e419
 LibOSXUnwind.v0.0.6+1.x86_64-apple-darwin.tar.gz/md5/fd98df2005d13aa16341c5aecba1af70
 LibOSXUnwind.v0.0.6+1.x86_64-apple-darwin.tar.gz/sha512/2d2263c3e5f095ad9eba7fea7cb882a19fece8a10486489d0a15b8e81ea0f8626804c4822b8c92a26a608d568c0ff1a4e976ea6d746be47ac1a70891455162a6
 LibUnwind.v1.3.2+4.aarch64-linux-gnu.tar.gz/md5/8f5cbf9820033211513f6d33e36194f1
```

Comment:
The LibOSXUnwind entries should be removed now that https://github.com/JuliaLang/julia/pull/39127 is merged